File: desafio-02/arlima/python/desafio2.py
"""
Desafio 2 - Adriano Roberto de Lima
"""

def main():
    """
    Main Function
    """

    for i in range(1, 10000):
        isprime = 1
        for j in range(2, i-1):
            if i % j == 0:
                isprime = 0
                break
        if isprime:
            print(i)

if __name__ == "__main__":
    main()

File: desafio-02/TeijiW/java/desafio_2.java
public class desafio_2 {
    public static void main(String[] args) {
        for (var i = 1; i < 10000; i++) {
            var is_prime = true;
            for (var j = 2; j < i; j++) {
                if (i % j == 0 && i != j) {
                    is_prime = false;
                    break;
                }
            }
            if (is_prime)
                System.out.println(i);
        }
    }
}